modules/ols-transcript-collection-overview.adoc

```diff
@@ -0,0 +1,18 @@
+// This module is used in the following assemblies:
+// about/ols-about-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: Concept
+[id="ols-transcript-collection-overview_{context}"]
+= Transcript collection overview 
+
+{ols-long} temporarily logs and stores complete transcripts of conversations that users have with the virtual assistant. This includes:
+
+* Queries from the user.
+
+* The complete message sent to the configured LLM provider, which includes system instructions, referenced documentation, and the user question.
+
+* The complete response from the LLM provider.
+
+Transcripts are associated with the cluster from which they originated, and {red-hat} can attribute specific clusters to specific customer accounts. Transcripts do not contain any information about which user submitted a question, and cannot be tied to any individual user.
+
+Transcripts are sent to {red-hat} every two hours, by default. If you are using the filtering and redaction functionality, the filtered or redacted content is sent to {red-hat}. {red-hat} will not see the original non-redacted content, and the redaction takes place before any content is captured in logs.
```

Comment:
I believe it could be beneficial to users if you shift this information above line no 8, i.e after the heading. Up to you though!

Reply:
done